week-2/debug/1.js

```diff
@@ -4,5 +4,5 @@ function sum(a, b) {
   return;
   a + b;
 }
-
 console.log(`The sum of 10 and 32 is ${sum(10, 32)}`);
+// the console won't run the code because the return function was not placed with the argument like this return a + b;
```

Comment:
Don't forget to update the function too! Also be careful with your language - `return` isn't a function. It's a keyword